pkg/util/log/stdout_logger.go

```diff
@@ -265,10 +274,17 @@ func (s *stdoutLogger) Fatalf(format string, args ...interface{}) {
 	s.logMutex.Lock()
 	defer s.logMutex.Unlock()
 
-	s.writeMessage(fatalFn, fmt.Sprintf(format, args...)+"\n")
+	msg := fmt.Sprintf(format, args...)
+
+	s.writeMessage(fatalFn, msg+"\n")
 	s.writeMessageToFileLoggerf(fatalFn, format, args...)
 
 	if s.fileLogger == nil {
+		analytics, err := analytics.GetAnalytics()
+		if err != nil {
```

Comment:
This check should be err == nil, otherwise send command event is never send or leads to a potential nil pointer